packages/core/src/configuration.ts

```diff
@@ -79,7 +79,7 @@ export const baseTooltip: TooltipOptions = {
 export const axisChartTooltip: AxisTooltipOptions = Tools.merge({}, baseTooltip, {
 	gridline: {
 		enabled: true,
-		threshold: 0.25
+		threshold: 0.02
```

Comment:
the current threshold I see on any datapoints seems to be too small. can we increase that please?

Reply:
Do you mean the ruler threshold or the gridline threshold? The reason we made the ruler threshold value so small is to prevent overlapping trigger events on datapoints which are really close to each other.